crates/wash-cli/src/up/mod.rs

```diff
@@ -272,6 +272,10 @@ pub struct WasmcloudOpts {
     /// If enabled, allows starting additional wasmCloud hosts on this machine
     #[clap(long = "multi-local")]
     pub multi_local: bool,
+
+    /// Defines the Max Execution time (in minutes or seconds) that the host runtime will execute for
+    #[clap(long = "max-execution-time", alias = "max-time", env = WASMCLOUD_MAX_EXECUTION_TIME)]
+    pub max_execution_time: Option<String>,
```

Comment:
Let's set the default here and remove the `Option`
```suggestion
    /// Defines the Max Execution time (in minutes or seconds) that the host runtime will execute for
    #[clap(long = "max-execution-time", alias = "max-time", env = WASMCLOUD_MAX_EXECUTION_TIME, default_value = DEFAULT_MAX_EXECUTION_TIME)]
    pub max_execution_time: String,
```

ideally, though, this would be a duration type like the one provided by https://docs.rs/clap-duration/0.1.11/clap_duration/macro.duration_range_value_parse.html